builds/jenkins-build-pack-website-azure/Jenkinsfile

```diff
@@ -0,0 +1,877 @@
+		} else {
+			error "Service Id is not available."
+		}
+	}
+
+	if (!service_config) {
+		error "Failed to fetch service metadata from catalog"
+	}
+
+	environmentDeploymentMetadata.initialize(service_config, configLoader, scmModule, branch, env.BUILD_URL, env.BUILD_ID, g_base_url + "/jazz/environments", auth_token)
+
+	def environment_logical_id = environmentDeploymentMetadata.getEnvironmentLogicalId();
+	if (params.scm_branch == 'master') {
+		environment = 'prod'
+		current_environment = 'PROD'
+		s3Bucket = configLoader.JAZZ.S3.WEBSITE_PROD_BUCKET;
+	} else {
+		if (environment_logical_id) {
+			environment = environment_logical_id
+		} else {
+			error "The environment has not been created yet and its missing the logical Id"
+		}
+		current_environment = "DEV"
+		s3Bucket = configLoader.JAZZ.S3.WEBSITE_DEV_BUCKET;
+	}
+
+	if (!events) { error "Can't load events module"	} //Fail here
+	events.initialize(configLoader, service_config, "SERVICE_DEPLOYMENT", branch, environment, g_base_url + "/jazz/events")
+
+	def create_cloudfront_url
+	context_map = [created_by : service_config['created_by']]
+	scmModule.setServiceConfig(service_config)
+	gitCommitHash = scmModule.getRepoCommitHash()
+	gitCommitOwner = scmModule.getRepoCommitterInfo(gitCommitHash)
+
+	def requestId = utilModule.generateRequestId()
+	if (requestId != null) {
+		events.setRequestId(requestId)
+		environmentDeploymentMetadata.setRequestId(requestId)
+	} else {
+		error "Request Id Generation failed"
+	}
+
+	dir(repo_name)
+	{
+		events.sendStartedEvent('CREATE_DEPLOYMENT', 'Website deployment started', environmentDeploymentMetadata.generateDeploymentMap("started", environment, , gitCommitHash), environment)
+
+		stage('Pre-Build Validation'){
+
+			events.sendStartedEvent("VALIDATE_INPUT", 'pre-build validation starts', context_map)
+			try {
+				service = service_config['service']
+				domain = service_config['domain']
+				if (domain != '' && domain != null) {
+					service = domain + '-' + service
+				}
+
+				create_cloudfront_url = true
+				if (service_config['create_cloudfront_url'] == true || service_config['create_cloudfront_url'] == "true") {
+					create_cloudfront_url = true
+				}
+
+				echo "create_cloudfront_url====" + create_cloudfront_url
+				send_status_email('STARTED', '')
+				validateTemplate()
+				validateDeploymentConfigurations()
+			} catch (ex) {
+				send_status_email('FAILED', '')
+				events.sendFailureEvent('VALIDATE_PRE_BUILD_CONF', ex.getMessage(), context_map)
+				events.sendFailureEvent('UPDATE_DEPLOYMENT', ex.getMessage(), environmentDeploymentMetadata.generateDeploymentMap("failed", environment, gitCommitHash), environment)
+				error ex.getMessage()
+			}
+			events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', 'pre-build validation completed', context_map)
+		}
+
+		stage("Deployment to ${current_environment} Env"){
+			events.sendStartedEvent("DEPLOY_TO_AWS", 'deployment starts', context_map)
```

Comment:
Why does the Azure build pack have code for AWS? Looks like this is a copy paste from API build back, so please clean it up before submitting it for PR.

Reply:
It was a mistake to have the Azure build pack in it. It is no need. The azure code has been built  in the current website build pack. Will remove it from the PR